website/project/presentation.rst

```diff
@@ -37,37 +22,58 @@ non-trivial implementation details (like the use of external libraries).
 You should also take care to explain the main dependencies your code has
 with the rest of chiventure.
 
-Please note that you should not need to show any of your code during
-your presentation: you should instead focus on describing at a high level
-the features you added or improved in chiventure, including any modules, data
-structures, etc. you implemented. That said, it can be
+Please note that this section of the report should **not** be a comprehensive description
+of every single file and function of your code. Instead, you should focus
+on describing **at a high level** the features you added or improved in chiventure,
+including any modules, data structures, etc. you implemented.
+
+A good way
+to think about this section is to imagine that it will be read
+by students in following years who will continue working on
+your feature. When writing this part of the report, ask yourself what information
+would be useful to future students, and assume that they have
+the technical skills to read through the code and fill in
+all the lower-level details of your work (but that they
+need a good sense of the "big picture" before doing so).
+
+For example, if you added a new data structure called
+``foo_t``, it would be appropriate to mention that you
+did so, and to describe the purpose of that data structure,
+as well as any dependencies related to that data structure.
+On the other hand, you would not include the full declaration
+of that data structure, nor would you describe each individual
+field. Instead, the goal is to convey that ``foo_t`` is a data
+structure relevant to your feature, and that anyone working
+on that feature in the future will need to familiarize themselves
+in it (and the assumption is that those students will just look
+at the code to learn more about the specifics of that data structure).
+
+Along those lines, you should not need to include any code
+in this section of the report, although it can be
 appropriate to show very short examples (3-4 lines) of how your code
 might be called from other components.
 
-Demo
-~~~~
+There is no specific length requirement for this section, but we
+expect any sufficiently-detailed section to be at least 2-3 pages long.
```

Comment:
Students will probably ask if this is single or double-spaced. Also, since it's somewhat difficult to assess page count when writing text or markdown files, would we want to give a rough range for number of words (e.g. ~400 words per page double spaced, so ~800-1200 words)?